tests/ctst/common/hooks.ts

```diff
@@ -54,7 +54,13 @@ After(async function (this: Zenko, results) {
     );
 });
 
-After({ tags: '@Quotas' }, async function () {
+After({ tags: '@Quotas' }, async function (this: Zenko, results) {
+    if (results.result?.status === 'FAILED') {
+        this.logger.warn('quota was not cleaned for test', {
+            bucket: this.getSaved<string>('bucketName'),
+        });
+        return;
+    }
```

Comment:
should this not be done in `teardownQuotaScenarios` ?

Reply:
I don't think so, we don't need to parse the results from cucumberjs in this function, we want to teardown when applicable, similar to "cleanS3Bucket"